.github/workflows/on-push-master.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will push a release to NPM
+
+name: On push master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  version-package:
+    name: version-package
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Setup Node.js 16
```

Comment:
Switch to nvm here too?

Will be nice to resolve these `name: Setup Node.js 16` with `node-version: 14` issues for good.  😉